### DISCUSSION TITLE
When using useServerSeoMeta for prerendered site, metadata does not update on navigation

### DISCUSSION THREAD

**Discussion by dbzx10299** [nuxt/nuxt#20812] [Questions]
I have a simple site that is pre rendered, and when I look at the output folder I can see all the appropriate metadata is in each generated html file.

However if I run `npm run preview` and look at the devtools, I can see that on navigation the metadata does not update.

I am using `useServerSeoMeta`. If I am on the homepage, with title: 'home' and navigate to the contact page with the title: 'contact', the title is still 'home' on the contact page until I refresh the page. I am a bit confused as to why this is happening since the contact page has all of the right information in it in the `.output` folder.

The concern I have is on a blog post, if someone navigates to a different page, and the server seo metadata does not update, then if the user shares the post then the information will be wrong.

I know it was recommended to use `useServerSeoMeta` instead of `useSeoMeta` but why won't the prerendered site update the metadata on navigation?

**Comment by oneminch** [comment]
I'm experiencing the same issue. But upon close inspection of the docs, `useServerSeoMeta` doesn't do anything on the client-side. It's recommended for metadata that isn't normally reactive, like `robots` or OpenGraph fields. You can define metadata you don't need on the client-side using `useServerSeoMeta` and metadata that changes from page to page using `useSeoMeta`. I don't know if that's officially recommended, but it makes logical sense.

```js
const title = ref('My title')

useSeoMeta({
  title: () => title,
  description: () => `description: ${title.value}`
})

useServerSeoMeta({
  ogTitle: () => title,
  ogDescription: () => `description: ${title.value}`,
  robots: "index, follow"
})
```
